apps/src/StudioApp.js

```diff
@@ -2030,6 +2030,10 @@ StudioApp.prototype.configureDom = function(config) {
     }
   }
 
+  // This prevents the app from extending offscreen when opening/closing the
+  // toolbox (if the app has a collapsible toolbox).
+  container.style.overflow = 'hidden';
```

Comment:
Where does this style get cleared? (so dropdowns don't get clipped after switching to text mode and back)

Reply:
Can you clarify?

Reply:
Followed up offline. My understanding is that there are two different wrappers. We no longer clip on the inner wrapper, but still clip on the outer wrapper. That's probably ok, because users would need a very small screen size to trigger this remaining bug:

![image](https://user-images.githubusercontent.com/413693/61402897-ee631400-a888-11e9-9e67-1eed57a7189d.png)

Reply:
I also don't think I'd consider it a bug. It makes sense for a dropdown like that to display over the instruction space, but when it starts displaying over the nav bar, it starts getting in the way of different types of user actions. i.e. when I'm coding, it makes sense that some items might overflow into the instructions. When I'm trying to go to a different lesson, it doesn't make sense for items from the coding workspace to overflow on the level progression. i.e. the app should be contained by the project space, not displayed on top of the project space